data/pirate missions smuggling intro.txt

```diff
@@ -0,0 +1,1174 @@
+					goto disparity
+				`	"BURN THE BANKS!"`
+					goto burn
+				`	"I guess. I don't really have an opinion."`
+					goto guess
+			`	The woman appears genuinely surprised. "You might be right. But don't we have one of those excesses right here? How long did you have to dream about this moment before it became a reality? How hard did you have to work? And now they're breaking your back for more - and interest! It's disgusting. Shouldn't there be a middle ground?"`
+				goto meetup3
+			label guess
+			`	The woman stares thoughtfully. "Temperance is a wonderful thing to have. But I wonder if that kind of uncertainty is what allows banks to exploit new pilots with so-called 'financial aid'."`
+				goto meetup3
+			label burn
+			`	The woman laughs, then in a raucous tone responds, "A bonfire of banknotes! And kindling of the bankers! All debts forgiven!"`
+				goto meetup3
+			label disparity
+			`	The woman nods. "That's a surprisingly detailed criticism. Most people who disagree with the private finance industry are far less eloquent. I think it might serve you well, once you get out from under your current obligation to it."`
+			
+			label meetup3
+			`	You feel a small pang of nausea at the mention of your own outstanding debt. The woman offers a look of pity and understanding. "I'm sorry. No matter how you look at it, it's just not fair."`
+			`	The woman looks around for a moment, then exclaims, "Oh! But I had a reason for all this, and believe it or not, it's related. Now that I've taken up so much of your time, let me introduce myself - my name is Alexandra Merrin. May I ask your name?"`
+			choice
+				`	"I'm <first> <last>."`
+				`	"My name is <first> <last>, at your service, my dear."`
+				`	"Sorry, I don't really have time for this."`
+					goto endno
+			`	"It's a pleasure to meet you, <first>."`
+				goto meetup4
+			
+			label meetup4
+			`	"I'm a scout for a special courier firm. Basically, we look for new captains who bought their ships in debt, and are struggling to make due. We front enough money to forgive their debt in return for a few deliveries, passenger runs, you get the gist. Sort of like a network of amateur couriers spread across the Republic."`
+			choice
+				`	"That sounds a lot like spaceport job postings. What's the difference?"`
+					goto postings
+				`	"Forgive their debt? Sure, I've heard that one before."`
+					goto forgive
+			
+			label postings
+			`	Alexandra smiles knowingly. "The difference is, we have a reliable courier, so we can cut expenses, and you get paid up front, so you can pay off your debt immediately - no loan interest spoiling every delivery you make. I mean, if you took out a 500,000 credit loan to buy your ship, that's over 295,000 credits in interest by the end of the year. And all that assuming the bank gave you a 'fair' 0.3% daily interest rate."`

```

Comment:
I would rewrite this sentence as follows:

`"The difference is, we have reliable couriers, so we can cut expenses. We pay the difference to couriers like you, so you can get out of debt immediately - no loan interest spoiling every delivery you make`